src/Controller/CaseReminderController.php

```diff
@@ -41,7 +38,24 @@ public function __construct(EntityManagerInterface $entityManager, ReminderHelpe
     }
 
     /**
-     * @Route("/new", name="reminder_create", methods={"POST"})
+     * @Route("/reminder", name="reminder_index")
+     */
+    public function index(ReminderRepository $reminderRepository): Response
+    {
+        /** @var User $user */
+        $user = $this->security->getUser();
+
+        $reminders = $reminderRepository->findBy([
+            'createdBy' => $user->getId()->toBinary(),
+        ]);
+
+        return $this->render('reminder/index.html.twig', [
+            'reminders' => $reminders,
+        ]);
+    }
+
+    /**
+     * @Route("/case/{id}/reminder/new", name="reminder_create", methods={"POST"})
```

Comment:
Consider renaming the route name to reminder_new to conform to the method name.

Reply:
Renamed route to `reminder_new`.